docs/data/material/all-components/all-components.md

```diff
@@ -0,0 +1,37 @@
+# Material UI components
+
+<p class="description">Every Material UI component available so far.</p>
+
+While we strive to follow the Material Design guidelines where practical (applying common sense where guidelines contradict - a more common occurrence than one might expect), we do not expect to support every component, nor every feature of every component, but rather to provide the building blocks to allow developers to create compelling user interfaces and experiences.
```

Comment:
This reads a little "negative" IMO. I would start with the goal which explains the rest of the paragraph:

```suggestion
We aim to provide the building blocks for developers to create compelling user interfaces and experiences.
Because of that we strive to follow the Material Design guidelines as closely as it's practical, applying common sense where guidelines contradict, but we do not expect to support every component or feature of every component.
```

Reply:
Good call, yeah! I copied this paragraph from the "Supported components" page, but definitely agree that it could use some rephrasing there. 🤙